variables.tf

```diff
@@ -140,12 +144,30 @@ variable "highly_available" {
   default     = false
 }
 
+variable "highly_available_read_replica" {
+  description = "Whether the MySQL instance should be highly available (REGIONAL) or not (ZONAL). Highly Available (HA) instances will automatically failover to another zone within the region if there is an outage of the primary zone. HA instances are recommended for production use-cases and, therefore, increase cost"
+  type        = bool
+  default     = false
+}
+
 variable "read_replica_count" {
   description = "Specify the number of read replicas for the MySQL instance. Value greater than 0 requires 'var.pit_recovery_enabled' to be 'true'."
   type        = number
   default     = 0
 }
 
+variable "allocated_ip_range_read_replica" {
+  description = <<-EOT
+  The name of the allocated ip range for the private ip CloudSQL read replica instance. 
+  If already available this can be derived from VPC -> Private Service Connection -> Name of the allocated range
+  For example: "google-managed-services-default". If set, the instance ip will be created in the allocated range. 
+  The range name must comply with RFC 1035. Specifically, the name must be 1-63 characters long and match the regular expression a-z?. 
+  If set, a CIDR range of /20 is advised.
```

Comment:
```suggestion
  The name of the allocated IP range for the private IP of read replicas of the CloudSQL instance. 
  If already available this can be derived from VPC -> Private Service Connection -> Name of the allocated range. For example: "google-managed-services-default".
  If set, the IP will be created within the allocated IP range. 
  The range name must comply with RFC 1035. Specifically, the name must be 1-63 characters long and match the regular expression a-z?. 
  If set, a CIDR range of /20 is advised.
```